src/libsyntax/parse/parser.rs

```diff
@@ -2839,6 +2839,17 @@ impl<'a> Parser<'a> {
                 let (span, e) = self.interpolated_or_expr_span(e)?;
                 (lo.to(span), ExprKind::AddrOf(m, e))
             }
+            token::Ident(..) if self.token.is_keyword(keywords::In) => {
```

Comment:
Wat.
I thought we removed the `in` version, like, years ago.

Reply:
Nope, it was removed in @aidanhs's PR